### ISSUE TITLE
[T12][PowerChart][Functional][Accessibility] Focus is missing on Calender drop down in Add condition window in Past Medical Tab

### ISSUE THREAD

**Issue by shiva87** [Mohanme13/Feb-19-Bug-bash#145]
Steps to reproduce:
1. Open Powerchart to test, testing patient.
2. Navigate to Histories tab to past medical tab,
3. Try to focus on calendar drop down for onset and resolved date fields.
